src/SearchBar.js

```diff
@@ -6,9 +6,17 @@ import locateImage from './img/icon_512x512.png';
 
 
 export function SearchBar(props){
+
   const [input, setInput] = useState(props.currZip || '');
   const[rangeInput, setRangeInput] = useState(props.currRange);
 
+  // frm: filters what events are displayed according to the kind of event
+  const [eventKindInput, setEventKindInput] = useState(props.currEventKind || 'ALLEVENTS');      
+   
+  // frm: filters what events are displayed according to the date ranges for the events
+  // frm: NYI - I will implement this once folks are happy with filtering on kind of event
+  const [dateRange, setDateRange] = useState({start: "now", end: "now+14"});
```

Comment:
Might make sense to keep this out for this PR to keep it small and get it merged in sooner?

Reply:
I removed dateRange and setDateRange.  Will add them back later when/if I add filtering by date range